HousingSearchListener/HousingSearchListener.csproj

```diff
@@ -28,7 +28,7 @@
     <PackageReference Include="Microsoft.Extensions.Logging" Version="5.0.0" />
     <PackageReference Include="Microsoft.Extensions.Logging.Abstractions" Version="5.0.0" />
     <PackageReference Include="Microsoft.Extensions.Logging.Configuration" Version="5.0.0" />
-    <PackageReference Include="NEST" Version="7.13.0" />
+    <PackageReference Include="NEST" Version="7.11.1" />
```

Comment:
why did you downgrade?

Reply:
is automatic update. it revert back to the previous version